app/middleware/renderer.js

```diff
@@ -1,8 +1,8 @@
 const log = require('../lib/logger');
 const mapLink = require('../lib/mapLink');
 
-function notFound(req, res, msg) {
-  log.warn(msg, 'Not found (404)');
+function notFound(req, res) {
+  log.warn({ req }, 'Not found (404)');
```

Comment:
I know it's already been merged, but I'm happy for the whole request to be logged, as long as it's not every request. So this seems sensible to me.